html/semantics/forms/autofocus/supported-elements.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/utils.js"></script>
+<script>
+"use strict";
+
+promise_test(async t => {
+  let w = window.open('/common/blank.html');
+  await waitForLoad(w);
+  w.document.body.innerHTML = '<div contenteditable=true autofocus></div>';
+  await waitUntilStableAutofocusState(t);
+  assert_equals(w.document.activeElement.tagName, 'DIV');
+  w.close();
```

Comment:
This would be better inside a `t.add_cleanup`, so that if the assertion fails, the window still closes.

Reply:
Done.